boards/px4/fmu-v3/src/spi.cpp

```diff
@@ -31,6 +31,42 @@
  *
  ****************************************************************************/
 
+#include <nuttx/spi/spi.h>
+#include <px4_arch/spi_hw_description.h>
+#include <drivers/drv_sensor.h>
+
+constexpr px4_spi_bus_t px4_spi_buses[SPI_BUS_MAX_BUS_ITEMS] = {
+	initSPIBus(1, {
+		// Cube 2.0
+		initSPIDevice(DRV_GYR_DEVTYPE_MPU6000, SPI::CS{GPIO::PortC, GPIO::Pin2}),
+
+		// Cube 2.1
+		initSPIDevice(DRV_GYR_DEVTYPE_MPU9250, SPI::CS{GPIO::PortC, GPIO::Pin2}),
```

Comment:
@bkueng Can you bake in the HW version here and simplify the select logic?

Reply:
I added the versions in https://github.com/PX4/Firmware/pull/14156/commits/c3531afde9e77510ecbc784ae6c9ee6789d13d98. This makes it indeed much cleaner and allows to use the generic chip-select code (not done yet).